site/src/sections/Navbar/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import pattern from "../../assets/images/brand/SVG/service-mesh-patterns-side.svg";
+import { Link } from "gatsby";
+import Layer5 from "../../assets/images/brand/SVG/Layer5.svg";
+import { Navigation, Toggle , Navbox , Hamburger , LogoWrap , NavItem } from "./navbar.style";
+import Button from "../../components/Button";
+
+const Navbar1 = () => {
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+  return (
+    <Navigation>
+      <LogoWrap as={Link} to="/">
+        <img className="pattern" src={pattern} alt="pattern" />
+      </LogoWrap>
+      <Toggle
+        navbarOpen={navbarOpen}
+        onClick={() => setNavbarOpen(!navbarOpen)}
+      >
+        {navbarOpen ? <Hamburger open /> : <Hamburger />}
+      </Toggle>
+      {navbarOpen ? (
+        <Navbox>
+          <NavItem to="/">Book Chapters</NavItem>
+          <NavItem to="/">Meet the Authors</NavItem>
+          <NavItem to="/">Patterns in action</NavItem>
+          <NavItem to="/about">About <img className="layer5" src={Layer5} alt="Layer5 logo" /></NavItem>
+          <NavItem to="/get"><Button><span className="label">Get it Now</span></Button></NavItem>
+        </Navbox>
+      ) : (
+        <Navbox open>
+          <NavItem to="/">Book Chapters</NavItem>
+          <NavItem to="/">Meet the Authors</NavItem>
+          <NavItem to="/">Patterns in action</NavItem>
+          <NavItem to="/about">About <img className="layer5" src={Layer5} alt="Layer5 logo" /></NavItem>
+          <NavItem to="/get"><Button><span className="label">Get it Now</span></Button></NavItem>
```

Comment:
I don't think the CSS for the `button` and the extra `span` is required anymore.
The `Button` component provides them all. Also, for the button text use the `title` prop that we have in the `Button` component. For reference either check the component code or layer5.io repo

Reply:
I have removed the ```span``` but kept the custom css as the default button css are not optimum 

![image](https://user-images.githubusercontent.com/95926324/190986511-e123aceb-b043-4aa7-8a5f-0b70601b167f.png)

Reply:
Well, you need to provide the `secondary` prop 🤷🏼‍♂️